internal/neofs/tree.go

```diff
@@ -1320,7 +1334,6 @@ func metaFromMultipart(info *data.MultipartInfo, fileName string) map[string]str
 	info.Meta[uploadIDKV] = info.UploadID
```

Comment:
Can the first object ID be used for UploadID?

Reply:
So you can upload the first object (even with 0 payload!) and use its hash, right?

Reply:
No, any part must have at least 5Mb

Reply:
I mean NeoFS side. The first object can be created during upload init.

Reply:
Well, in this case In theory - yes. However, UploadID it just a `uuid` string. It is simple to generate.
Trying the approach with first object id as an UploadID requires many code changes, extended control for this _zero_ object during uploading and completing.
I see many problems and no profit for changing it right now.

Reply:
1. Per S3 spec UploadID is just a string. It can be anything, including OID.
2. Client will use this UploadID for _every_ subsequent request which means you'll get this OID directly, you do not need to map UUID to first OID.
3. The first object would then be _always_ available, even if you're reuploading the first chunk.
4. There is nothing to control there, just create it for `CreateMultipartUpload`. Nothing you need to do with it afterwards (except for regular usage as first object OID).

Reply:
Do we do this inside this PR or in a separate issue?

Reply:
It's a part of v2 switch to me. It was specifically designed to allow for it.

Reply:
Updated